<launch>
    <node name="animation_render" pkg="animation_render" type="animation_render_node" output="screen">
        <param name="control_topic" value="/tplsearch/control"/>

        <param name="autostart" value="0"/>

        <param name="render_filename" value="cylinder"/>
        <param name="videos"    value="/home/sebastian/videos/"/>
        <param name="templates" value="/home/sebastian/templates/"/>

        <param name="object" value="Cylinder"/>
        <!-- Length, Height -->
        <param name="model_parameter_1" value="0.05"/>
        <!-- Radius, Width -->
        <param name="model_parameter_2" value="0.01"/>
        <!-- Faces -->
        <param name="model_parameter_3" value="200"/>

        <!--
        RotationX RotationY RotationZ
        TranslationZ
        -->
        <param name="animation" value="RotationY"/>


        <param name="template_list_length"    value="100"/>
        <param name="background_list_length"  value="1"/>
        <param name="template_keywords"       value="book"/>
        <param name="background_keywords"     value="room"/>

        <rosparam command="load" file="$(find animation_render)/launch/video_params.yaml" />

        <param name="skip_template_download"   value="true"/>
        <param name="skip_background_download" value="true"/>
        <param name="skip_ground_truth"        value="true"/>
        <param name="skip_render"              value="true"/>
        <param name="skip_img_list"            value="true"/>

        <param name="template_resize" value="100"/>
        <param name="threshold_acceptance" value="0.1"/>
        <param name="threshold_gradient"   value="30"/>
    </node>
</launch>
